source/code/go/src/plugins/oms.go

```diff
@@ -734,9 +741,22 @@ func PostDataHelper(tailPluginRecords []map[interface{}]interface{}) int {
 
 		stringMap := make(map[string]string)
 
-		stringMap["LogEntry"] = ToString(record["log"])
+		logEntry := ToString(record["log"])
+		logEntryTimeStamp := ToString(record["time"])
+		if isDockerContainerRuntimeEngine == false {
+			//CRI compatiable runtimes uses the CRIO parser which has logtag followed space and then log line
+			// so trimming the log tag (P or F) 
+			logEntry = strings.TrimSpace(logEntry)
+			if strings.HasPrefix(logEntry, "P ") {
+			   logEntry = strings.TrimPrefix(logEntry, "P ")
```

Comment:
This will be an expensive operation to trim on every single logline. So basically we were blindly flushing the logline before and now we are processing & trimming every single log line. Are you sure that this ("P ") is passed-in to us in the same logEntry field ?

Reply:
since the regex doing this stuff so removed this logic.